DNA-RNA.py

```diff
@@ -0,0 +1,58 @@
+seq = str(input()).upper()
+
+
+class RNA:
```

Comment:
Class names should normally use the CapWords convention (not the UPPERCASE)

Reply:
Uppercase is allowed if it's abbreviation. 
According to PEP-8:
> Note: When using abbreviations in CapWords, capitalize all the letters of the abbreviation. Thus HTTPServerError is better than HttpServerError.